packages/converters/src/to-boolean/index.ts

```diff
@@ -10,30 +10,33 @@
  * governing permissions and limitations under the License.
  */
 
-import { isTrue } from '@accelint/predicates';
-
 /**
- * Compare the given value against a custom list of `truthy` values.
- *
- * String values are not case sensitive.
+ * Will consider any non "false-value" to be true. String values are not case sensitive.
  *
- * _1, '1', 'on', 'true', 'yes', true_
+ * **"false-values"**
+ *   - inherently false values: '' (empty string), 0, false, undefined, null, NaN
+ *   - numeric zero: '0.000' - any number of leading or trailing zeros
+ *   - string literal: 'false' - any capitalizations or space-padding
  *
  * @pure
  *
  * @example
- * toBoolean('on');
- * // true
- *
- * toBoolean('yes');
+ * toBoolean(1);
  * // true
  *
- * toBoolean('off');
+ * toBoolean(' FaLsE ');
  * // false
  *
- * toBoolean('no');
+ * toBoolean('  true');
+ * // true
+ *
+ * toBoolean('000.000');
  * // false
  */
 export function toBoolean(val: unknown) {
-  return isTrue(val);
+  return !(
+    !val ||
+    `${val}`.trim().toLowerCase() === 'false' ||
+    Number.parseFloat(val as string) === 0
```

Comment:
Why not wrap it in a string template literal? Since `unknown` is definitely not `string`

Reply:
I think he meant `Number.parseFloat(val as string)` -> ``Number.parseFloat(`{val}`)``

Reply:
That is why I asked. :)

That makes more sense.

Reply:
I can, and have changed it, but I remember my original reasoning being that `as string` is only for typescript so it will be removed in production while wrapping in a string template would carry into production.

Reply:
I would agree with that though. `Number.parseFloat({}) === NaN` which doesn't `=== 0` so the logic checks out. Which I guess a case could be made, are we considering `NaN` as `false`?

Reply:
It is a good question. I asked myself that as well. What I fell back on is that Javascript considers is to be false so I felt that staying consistent with the language is best.